tests/test_heartbeat.py

```diff
@@ -162,7 +161,7 @@ def test_heartbeat_can_open_channel_client_close():
     print('SERVER LOST CLIENT :)')
     server_hbchan.close()
     server.close()
-
+    client.close()
```

Comment:
This test fails in python 3.9 if client is closed before server hb channel is closed. Looking at the other tests, it seemed to me that client.close() should be called after server.close().